drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -411,6 +417,10 @@ Error AudioDriverWASAPI::audio_device_init(AudioDeviceWASAPI *p_device, bool p_c
 
 		hr = device_audio_client_3->InitializeSharedAudioStream(0, period_frames, pwfex, nullptr);
 		ERR_FAIL_COND_V_MSG(hr != S_OK, ERR_CANT_OPEN, "WASAPI: InitializeSharedAudioStream failed with error 0x" + String::num_uint64(hr, 16) + ".");
+		uint32_t output_latency_in_frames;
+		WAVEFORMATEX *pwfex;
```

Comment:
Could you clean up this struct when you're done using it? It looks like [this will leak memory according to the documentation](https://docs.microsoft.com/en-us/windows/win32/api/audioclient/nf-audioclient-iaudioclient3-getcurrentsharedmodeengineperiod#remarks) unless you clean it up explicitly.

Reply:
Fixx'd